click-game/js/cookies.js

```diff
@@ -0,0 +1,29 @@
+var cookiesObj = {
+    setCookie: function(name,value,days) {
+        var expires = "; SameSite=None; Secure; expires=Thu, 2 Aug 2070 20:47:11; path=/";
+        if (days) {
+            var date = new Date();
+            date.setTime(date.getTime() + (days*24*60*60*1000));
+            expires = "; SameSite=None; Secure; expires=" + date.toUTCString();
+        }
+        document.cookie = name + "=" + (value || "")  + expires + "; path=/";
+    },
+    getCookie: function(name) {
+        var nameEQ = name + "=";
+        var ca = document.cookie.split(';');
+        for(var i=0;i < ca.length;i++) {
+            var c = ca[i];
+            while (c.charAt(0)==' ') c = c.substring(1,c.length);
+            if (c.indexOf(nameEQ) == 0) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use ===/!== to compare with true/false or Numbers](https://app.codacy.com/app/edazpotato/edazpotato.github.io/pullRequest?prid=4679772)